netbox/dcim/models.py

```diff
@@ -500,6 +537,21 @@ def __init__(self, *args, **kwargs):
     def get_absolute_url(self):
         return reverse('dcim:devicetype', args=[self.pk])
 
+    def to_csv(self):
+        return csv_format([
+            self.manufacturer,
+            self.model,
+            self.slug,
+            self.part_number,
+            self.u_height,
+            self.is_full_depth,
+            self.is_console_server,
+            self.is_pdu,
+            self.is_network_device,
+            self.get_subdevice_role_display() if self.subdevice_role else None,
+            self.get_interface_ordering_display() if self.interface_ordering else None,
```

Comment:
Just `self.get_interface_ordering_display()`: `interface_ordering` is a required field